docs/resources/Guild.md

```diff
@@ -290,6 +296,26 @@ Create a new guild. Returns a [guild](#DOCS_RESOURCES_GUILD/guild-object) object
 > info
 > If roles are specified, the required `id` field within each role object is an integer placeholder, and will be replaced by the API upon consumption. Its purpose is to allow you to [overwrite](#DOCS_RESOURCES_CHANNEL/overwrite-object) a role's permissions in a channel when also passing in channels with the channels array.
```

Comment:
nit: lets move these info blocks above the examples since it feels like the documentation structure is a bit jumpy here